frontend/src/ListItem/ListItem.js

```diff
@@ -0,0 +1,51 @@
+import React from "react";
+import { Link } from "react-router-dom";
+
+const getTitle = (post) => {
+  let title = post.title.split("\n")[0];
+
+  if (title.length > 20) {
+    return title.slice(0, 20);
+  }
+
+  return title;
+};
+
+const getDate = (post) => {
+  const date = new Date(post.published_at);
+  const options = {
+    weekday: "long",
+    year: "numeric",
+    month: "long",
+    day: "numeric",
+  };
+
+  return date.toLocaleString("en-US", options);
+};
+
+const getContent = (post) => {
+  let content = post.content;
+  
+  if (content.length > 200) {
+    return content.slice(0, 200) + "...";
+  } else {
+    return content;
+  }
+};
```

Comment:
Follow DRY principle (Dont Repeat  Yourself)
This is repetitive function, create a new helper file and put it there.

Reply:
Noted sir, made a new helper folder